src/renderer/components/wallet/settings/Settings.tsx

```diff
@@ -49,9 +51,12 @@ export const Settings: React.FC<Props> = (props): JSX.Element => {
     removeLedgerAddress,
     */
     removeAllLedgerAddress,
-    changeNetwork
+    changeNetwork,
+    phrase = O.none
```

Comment:
[nitpick] 

` phrase = O.none` -> `phrase: oPhrase = O.none` , so you can change `wallet_phrase` -> `phrase` few lines later. (we don't use `snake_case` naming)